README.md

```diff
@@ -26,8 +26,13 @@ _Since Bevy is in heavy development, there regularly are unpublished new feature
 You should keep the `credits` directory up to date. The release workflow automatically includes the directory in every build.
 
 ### Updating the icons
- 1. Replace `build/windows/icon.ico` (used for windows executable and as favicon for the web-builds)
- 2. Replace `build/macos/icon_1024x1024.png` with a `1024` times `1024` pixel png icon and run `create_icns.sh` (make sure to run the script inside the `macos` directory) - _Warning: sadly this seems to require a mac..._
+ 1. Replace `build/macos/icon_1024x1024.png` with a `1024` times `1024` pixel png icon and run `create_icns.sh` (make sure to run the script inside the `build/macos` directory) - _Warning: sadly this seems to require a mac..._
+ 2. Replace `build/windows/icon.ico` (used for windows executable and as favicon for the web-builds)
+    * On macOS you can create this by:
```

Comment:
I use GIMP for ico creation, which should work the same on all major platforms. Open the png file and export it as `.ico`.
If that works on macOS, I would prefer a short scentence pointing to GIMP over a platform specific workflow.